postgres-appliance/scripts/upload_pg_log_to_s3.py

```diff
@@ -53,9 +53,15 @@ def upload_to_s3(local_file_path):
 
     chunk_size = 52428800  # 50 MiB
     config = TransferConfig(multipart_threshold=chunk_size, multipart_chunksize=chunk_size)
+    tags = eval(os.getenv('LOG_S3_TAGS'))
+    s3_tags = {}
+    for key, value in tags.items():
+        s3_tags[key] = os.getenv(value)
+
+    s3_tags_str = "&".join(f"{key}={value}" for key, value in s3_tags.items())
```

Comment:
it seems that we won't use the intermediate variable `s3_tags` anywhere else, would it make sense to directly building the tag string in one step?

```
s3_tags_str = "&".join(f"{key}={os.getenv(value)}" for key, value in tags.items())
```

Reply:
sounds good to me